site/_posts/2021-04-23-announcing-fusionauth-1-26.md

```diff
@@ -0,0 +1,103 @@
+
+## An API to manage API keys
+
+In true Inception style, there is now an API to manage FusionAuth API keys. This was a community request last year. You can create, copy, update, and delete API keys. These keys are indistinguishable from API keys created in the user interface. 
+
+They can be restricted in a number of ways:
+
+* By tenant
+* By endpoint
+* By permission on an endpoint
+
+This feature allows interesting automation. For example, you can now generate API keys with very specific permissions, distribute and use them as needed, and then destroy them.
+
+## Other enhancements
+
+Some of the other enhancements included in this release are:
+
+* Support for FusionAuth acting as an SP for IdP initiated SAMLv2 SSO (this is a premium feature).
+* An extension to the user registration forms to allow for a user profile self service page (this is a premium feature).
+* A Prometheus endpoint for monitoring.
+* Support for licensing in air-gapped deployments.
+* The login success and failure webhook events now include the IP address.
+
+Also, there's a new themeable FusionAuth index page so when your users accidentally arrive at `auth.example.com`, they are not forwarded to the FusionAuth admin UI login page. (To proceed to that login page when using the default theme, click the lock in the upper right.)
+
+{% include _image.liquid src="/assets/img/blogs/release-1-26/themable-index-page.png" alt="The themable root page. No more users ending up on the login page." class="img-fluid" figure=false %}
+
+Like I said, this was a big release!
+
+## Bugs squashed
+
+There were a couple of bugs fixed, including:
+
+* A registration for an inactive application previously couldn't be deleted; now it can be.
+* There was spurious text on the FusionAuth admin UI screen in certain scenarios. This is now gone.
+
+## Upgrade at will
+
+We're in the process of documenting all of this, but the [release notes](/docs/v1/tech/release-notes/#version-1-26-0) serve as a good guide of the changes, fixes, and new features.
+
+If you'd like to upgrade your self-hosted FusionAuth instance, see our [upgrade guide](/docs/v1/tech/installation-guide/upgrade/). 
+
+If you have a FusionAuth Cloud deployment, proceed to the "Deployments" tab on your [account dashboard](https://account.fusionauth.io/account/deployment/){:target="_blank"} and upgrade your servers. If you have any questions about the upgrade, [please open a support ticket](https://account.fusionauth.io/account/support/){:target="_blank"}.
+
+Or, if we've piqued your interest and you'd like to use FusionAuth, [check out your options](/pricing/).
```

Comment:
Or (remove comma) if we've piqued your interest and you'd like to use FusionAuth -  [check out your options!](/pricing/).

preference-based 🙂